ladders.js

```diff
@@ -38,6 +38,28 @@ class BattleReady {
 		/** @type {number} */
 		this.time = Date.now();
 	}
+
+	/**
+	 * 
+	 * @param {BattleReady} first The first item to compare
+	 * @param {BattleReady} second The second item to compare
+	 */
+	static sameFormatAs(first, second) { 
+		return first.formatid === second.formatid;
```

Comment:
There are situations, I think, where I would not _strongly_ object to one-liner helper functions for readability reasons.

But I don't feel like `first.formatid === second.formatid` is unreadable enough to deserve a function.

Reply:
hmm